Jenkinsfile_nightly

```diff
@@ -15,5 +15,25 @@ def product = "et"
 def component = "sya"
 
 withNightlyPipeline(type, product, component) {
+    env.TEST_URL = params.URL_TO_TEST
+    env.TEST_E2E_URL = params.URL_TO_TEST
```

Comment:
can we remove TEST_E2E_URL as we have not used it

Reply:
will do in the next patchset